Tiltfile

```
@@ -383,3 +386,34 @@ if localnet_config["rest"]["enabled"]:
     print("REST enabled: " + str(localnet_config["rest"]["enabled"]))
     deployment_create("rest", image="davarski/go-rest-api-demo")
     k8s_resource("rest", labels=["data_nodes"], port_forwards=["10000"])
+
+### Pocketdex Shannon Indexer
+def load_pocketdex():
+    pocketdex_tilt = load_dynamic("../pocketdex/pocketdex.tilt")
+    pocketdex_tilt["pocketdex"]("../pocketdex",
+                                genesis_file_name="localnet.json",
+                                postgres_values_path="../pocketdex/k8s/postgres/values.yaml",
+                                indexer_values_path="./localnet/kubernetes/values-pocketdex-indexer.yaml",
+                                gql_engine_values_path="./localnet/kubernetes/values-pocketdex-gql-engine.yaml")
+
+if localnet_config["indexer"]["enabled"]:
+    # Check if sibling pocketdex repo exists.
+    pocketdex_repo_exists = local("[ -d '../pocketdex' ] && echo 'true' || echo 'false'")
+    print("POCKETDEX_REPO_EXISTS: ", str(pocketdex_repo_exists).strip())
+    if str(pocketdex_repo_exists).strip() == "false":
+        if localnet_config["indexer"]["clone_if_not_present"]:
+            print("Cloning pocketdex repo")
+            local("git clone https://github.com/pokt-network/pocketdex --branch chore/tilt ../pocketdex")
```

TODO: Remove `--branch chore/tilt` flag once https://github.com/pokt-network/pocketdex/pull/23 is merged.

```suggestion
            local("git clone https://github.com/pokt-network/pocketdex ../pocketdex")
```